docs/setup/independent/install-kubeadm.md

```diff
@@ -86,15 +86,16 @@ apt-get update
 apt-get install -y docker.io
 ```
 
-or install Docker CE 17.03 from Docker's repositories for Ubuntu or Debian:
+or install Docker CE 17.09 from Docker's repositories for Ubuntu or Debian:
 
 ```bash
 apt-get update && apt-get install -y curl apt-transport-https
 curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
-cat <<EOF >/etc/apt/sources.list.d/docker.list
-deb https://download.docker.com/linux/$(lsb_release -si | tr '[:upper:]' '[:lower:]') $(lsb_release -cs) stable
-EOF
-apt-get update && apt-get install -y docker-ce=$(apt-cache madison docker-ce | grep 17.03 | head -1 | awk '{print $3}')
+add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") \
```

Comment:
The original snippet didn't really mention `armhf` here, which is an option to install, so I'm not sure whether this snippet needs to be broken out into two to handle `amd64` or `armfh`.

Is there a compelling reason here for these docs to exist separate from the Docker instructions?

Reply:
I'd link to the docker instructions for completeness, but have the example here as well.

Reply:
Added a link below the code snippets section